spec/classes/accesslogin_spec.rb

```diff
@@ -20,66 +20,76 @@
     END
 
     it do
-      should contain_file('access_conf').with({
-        'ensure'  => 'file',
+      is_expected.to contain_file('access_conf').with({
+                                                        'ensure' => 'file',
```

Comment:
I've seen this happen before with pdk validate is executed. Will fix this and other occurrences.